{"status": "broken", "name": "test image detection", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "test image detection"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython2"}], "stop": 1633557571033, "historyId": "f05a01afd7bc2ab14faddaa39b6c432e", "start": 1633557571029, "steps": [{"status": "broken", "start": 1633557571031, "stop": 1633557571033, "name": "Given Run image detection algorithm and load images from a directory and find \"Bus\"", "statusDetails": {"message": "IOError: [Errno 2] No such file or directory: '../../input/object_detection_classes_coco.txt'\n", "trace": "  File \"/home/meenakshi/anaconda2/lib/python2.7/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/meenakshi/anaconda2/lib/python2.7/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"steps/teststeps.py\", line 206, in loadimageforclassification\n    with open('../../input/object_detection_classes_coco.txt', 'r') as f:\n"}}], "statusDetails": {"message": "IOError: [Errno 2] No such file or directory: '../../input/object_detection_classes_coco.txt'\n", "trace": "  File \"/home/meenakshi/anaconda2/lib/python2.7/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/meenakshi/anaconda2/lib/python2.7/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"steps/teststeps.py\", line 206, in loadimageforclassification\n    with open('../../input/object_detection_classes_coco.txt', 'r') as f:\n"}, "uuid": "5fb584d3-ec6d-4798-98dc-d6101682823f"}